frontend/src/components/form/hash-input.jsx

```diff
@@ -0,0 +1,63 @@
+import { createEffect, createSignal } from "solid-js";
+import { styled } from "solid-styled-components";
+import icon from "../../assets/copy-ico.svg";
+
+const HashInput = (props) => {
+  const [value, setValue] = createSignal(
+    props.defaultValue || `djkbfsdjkbfsdj`
+  );
+  const [isValue, setIsValue] = createSignal(Boolean(props.defaultValue));
```

Comment:
`props.defaultValue.length == 0` is more explicit way to achieve this

Reply:
done